mapswipe_workers/mapswipe_workers/generate_stats/generate_stats.py

```diff
@@ -1,8 +1,33 @@
+from typing import Optional, List
+import datetime as dt
+
+from mapswipe_workers import auth
 from mapswipe_workers.definitions import DATA_PATH, logger
 from mapswipe_workers.generate_stats import overall_stats, project_stats
 
 
-def generate_stats(project_id_list: list):
+def get_recent_projects(hours: int = 3):
+    """Get ids for projects when results have been submitted within the last x hours."""
+    pg_db = auth.postgresDB()
+    query_insert_results = """
+        select project_id
+        from results
+        where start_time >= %(timestamp)s
```

Comment:
You are right on this one. It took a long time (> 3 min) to run this query on the production results table, that didn't have an index on the the timestamp.

I added an index on the "timestamp" attribute in the results table and this could improve the performance a lot when querying for new results.